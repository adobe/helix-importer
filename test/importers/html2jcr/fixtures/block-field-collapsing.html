
<!DOCTYPE html>
<html>
  <head>
    <title>Sites Franklin Example</title>
    <link rel="canonical" href="https://main--xwalk-example--buuhuu.hlx.page/">
    <meta name="description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta property="og:title" content="Sites Franklin Example">
    <meta property="og:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta property="og:url" content="https://main--xwalk-example--buuhuu.hlx.page/">
    <meta property="og:image" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
    <meta property="og:image:secure_url" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:title" content="Sites Franklin Example">
    <meta name="twitter:description" content="Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit... There is no one who loves pain itself, who seeks after ...">
    <meta name="twitter:image" content="https://main--xwalk-example--buuhuu.hlx.page/media_134b848c697e9152f269fa3b79696b855040912ba.png?width=1200&#x26;format=pjpg&#x26;optimize=medium">
    <meta name="foobar" content="hello world">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src="/scripts/lib-franklin.js" type="module"></script>
    <script src="/scripts/scripts.js" type="module"></script>
    <link rel="stylesheet" href="/styles/styles.css">
  </head>
  <body>
    <header></header>
    <main>
      <div>
        <div class="teaser">
          <div>
            <div>
              <picture>
                <img src="/content/dam/teaser-background.png" alt="A group of people sitting on a stage & relaxing">
              </picture>
            </div>
          </div>
          <div>
            <div>
              <h2>Meet the Experts</h2>
            </div>
          </div>
          <div>
            <div>
              <p><em><a href="https://www.adobe.com" title="This is a Secondary Button">Secondary Button</a></em></p>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer></footer>
  </body>
</html>