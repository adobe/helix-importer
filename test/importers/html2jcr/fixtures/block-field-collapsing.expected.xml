<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" jcr:title="Sites Franklin Example" sling:resourceType="core/franklin/components/page/v1/page">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Teaser" image="/content/dam/teaser-background.png" imageAlt="A group of people sitting on a stage &amp; relaxing" title="Meet the Experts" titleType="h2" link="https://www.adobe.com" linkType="secondary" linkText="Secondary Button" linkTitle="This is a Secondary Button" model="teaser"/>
            </section>
        </root>
    </jcr:content>
</jcr:root>